File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.py
import sys

input = sys.stdin.readline
print = sys.stdout.writelines

my_dict = {}
n, m = map(int, input().split())
for _ in range(n):
    k, v = input().rstrip().split()
    my_dict[k] = v
for _ in range(m):
    k = input().rstrip()
    print(f"{my_dict.get(k)}\n")

File: 백준/Silver/17219. 비밀번호 찾기/비밀번호 찾기.swift
let nm = readLine()!.split(separator: " ").map { Int($0)! }
let n = nm[0], m = nm[1]

var dict = [String: String]()
var output = ""

for _ in 0..<n {
    let input = readLine()!.split(separator: " ").map { String($0) }
    let k = input[0], v = input[1]
    dict[k] = v
}
for _ in 0..<m {
    let input = readLine()!
    output += dict[input]! + "\n"
}
print(output, terminator: "")
